### ISSUE TITLE
HttpOnly on cookie in CookieCache

### ISSUE THREAD

**Issue by ThomasArdal** [SimpleAuthentication/SimpleAuthentication#216]
Any reason not to set `HttpOnly` to `true` in `CookieCache`?:

```csharp
var cookie = httpContext.Request.Cookies[Name] ?? new HttpCookie(Name) {HttpOnly = true};
```

**Comment by PureKrome** [comment]
Sorry for the late reply @ThomasArdal 

Um - nope. no reason besides ignorance. 😢

**Comment by ThomasArdal** [comment]
Is the repo dead or do you want a PR? (I've seen you argue that ASP.NET Identity and similar makes this library redundant. But in my opinion, there is still room for a simpler alternative like this)

**Comment by PureKrome** [comment]
Personally, I'm moving more over to the core stuff. Part of me does want to move this up to .NET Standard and then maybe leave it in maintenance mode after that.

Happy to accept a PR cause there's another bug reported yesterday so I might try and do another release into a .NS release.

**Comment by PureKrome** [comment]
Fixed in #219
